rules-reviewed/openshift/session.windup.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<ruleset id="session"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This is a simple ruleset for detecting usage of data storage in HTTP session objects when migrating an application to a cloud environment.
+        </description>
+        <dependencies>
+          <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+          <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="java"/>
+        <sourceTechnology id="java-ee"/>
+        <targetTechnology id="cloud-readiness"/>
+        <targetTechnology id="openshift"/>
+        <tag>clustering</tag>
+    </metadata>
+    <rules>
+        <rule id="session-00000">
+            <when>
+                <and>
+                    <file filename="web.xml"/>
+                    <not>
```

Comment:
I think that this needs to be a xpath rule rather than this. As is, this rule also matches files that do not have distributable.